example/viam_example_app/android/app/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
         applicationId "com.example.viam_example_app"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion 23
```

Comment:
Seems like hardcoding it is worse than setting it in the flutter env? What was the issue here?

Reply:
`flutter.minSdkVersion` is coming directly from flutters code, not something we set in the flutter env (thought oyu can go in and change it manually in your version of flutter). because we use packages with higher requirements we need to manually bump it in the projects that use our SDK.

```
Launching lib/main.dart on sdk gphone64 arm64 in debug mode...
/Users/clint/Documents/viam/viam-flutter-sdk/example/viam_example_app/android/app/src/debug/AndroidManifest.xml Error:
	uses-sdk:minSdkVersion 19 cannot be smaller than version 21 declared in library [:flutter_webrtc] /Users/clint/Documents/viam/viam-flutter-sdk/example/viam_example_app/build/flutter_webrtc/intermediates/merged_manifest/debug/AndroidManifest.xml as the library might be using APIs not available in 19
	Suggestion: use a compatible library with a minSdk of at most 19,
		or increase this project's minSdk version to at least 21,
		or use tools:overrideLibrary="com.cloudwebrtc.webrtc" to force usage (may lead to runtime failures)

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugMainManifest'.
> Manifest merger failed : uses-sdk:minSdkVersion 19 cannot be smaller than version 21 declared in library [:flutter_webrtc] /Users/clint/Documents/viam/viam-flutter-sdk/example/viam_example_app/build/flutter_webrtc/intermediates/merged_manifest/debug/AndroidManifest.xml as the library might be using APIs not available in 19
  	Suggestion: use a compatible library with a minSdk of at most 19,
  		or increase this project's minSdk version to at least 21,
  		or use tools:overrideLibrary="com.cloudwebrtc.webrtc" to force usage (may lead to runtime failures)

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s


┌─ Flutter Fix ───────────────────────────────────────────────────────────────────────────────────┐
│ The plugin flutter_webrtc requires a higher Android SDK version.                                │
│ Fix this issue by adding the following to the file                                              │
│ /Users/clint/Documents/viam/viam-flutter-sdk/example/viam_example_app/android/app/build.gradle: │
│ android {                                                                                       │
│   defaultConfig {                                                                               │
│     minSdkVersion 21                                                                            │
│   }                                                                                             │
│ }                                                                                               │
│                                                                                                 │
│ Following this change, your app will not be available to users running Android SDKs below 21.   │
│ Consider searching for a version of this plugin that supports these lower versions of the       │
│ Android SDK instead.                                                                            │
│ For more information, see:                                                                      │
│ https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration            │
└─────────────────────────────────────────────────────────────────────────────────────────────────┘
Exception: Gradle task assembleDebug failed with exit code 1


Exited (1).

```

not sure why we have 23 in the readMe, the error output here says 21 - we could bump it down in the readme and example projects too. (I just tested locally and 21 compiles fine).